tests/e2e/browser/複数選択.spec.ts

```diff
@@ -0,0 +1,150 @@
+import { test, expect, Page } from "@playwright/test";
+import { toggleSetting, navigateToMain } from "../navigators";
+
+test.beforeEach(async ({ page }) => {
+  const BASE_URL = "http://localhost:5173/#/home";
+  await page.setViewportSize({ width: 800, height: 600 });
+  await page.goto(BASE_URL);
+
+  await navigateToMain(page);
+  await page.waitForTimeout(100);
+  await toggleSetting(page, "複数選択");
+
+  await prepareAudioCells(page, 4);
+});
+
+const ctrlLike = process.platform === "darwin" ? "Meta" : "Control";
+/**
+ * アクティブなAudioCellと選択されているAudioCellを取得する。
+ * 戻り値のインデックスは1から始まる。（nth-childのインデックスと揃えるため）
+ */
+async function getSelectedStatus(page: Page): Promise<{
+  active: number;
+  selected: number[];
+}> {
+  const selectedAudioKeys = await page.evaluate(() => {
+    const audioCells = [...document.querySelectorAll(".audio-cell")];
+    let active: number | undefined;
+    const selected: number[] = [];
+    for (let i = 0; i < audioCells.length; i++) {
+      const audioCell = audioCells[i];
+      if (audioCell.classList.contains("active")) {
+        active = i + 1;
+      }
+      if (audioCell.classList.contains("selected")) {
+        selected.push(i + 1);
+      }
+    }
+
+    return { active, selected };
+  });
+  if (selectedAudioKeys.active === undefined) {
+    throw new Error("No active audio cell");
+  }
+  // return selectedAudioKeys; だと何故か型エラーを出すので、一旦オブジェクトを作って返す
+  // FIXME: なんとかする
+  return {
+    active: selectedAudioKeys.active,
+    selected: selectedAudioKeys.selected,
+  };
+}
+
+async function prepareAudioCells(page: Page, count: number) {
+  for (let i = 1; i < count; i++) {
```

Comment:
あれこれcount=1の時って1つも追加されない･･･？やりたいことはこうかも？
```suggestion
  for (let i = 0; i < count; i++) {
```

Reply:
デフォルトで1つ用意されてるのでcount=1の時は何も追加されないのが正解ですね

Reply:
なるほどです！
この関数は「AudioCellsをcount個にする」という感じでしょうか。
とすると元が1個じゃなかったらバグってそう･･･？
これを正確に実装しようとすると減らさないといけなくて大変なので、addにしてcount個増やす実装が分かりやすい気がします。

Reply:
addAudioCellsにしました。